.travis.yml

```diff
@@ -1,16 +1,20 @@
 language: go
 sudo: required
 
+services:
+  - docker
+
 go:
   - 1.7.x
   - master
 
 
 before_install:
   - sudo apt-get install realpath -y
+  - docker run -d --name=etcd -p 2379:2379 quay.io/coreos/etcd:v3.0.17
```

Comment:
i think it might be better to make this part of the integration test target so people can successfully run the integration tests locally (assuming they have docker available).